src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLSPOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer
+{
+    internal class RazorLSPOptions
+    {
+        public Trace Trace { get; set; }
+
```

Comment:
Hmm should this class be immutable?

Reply:
It can be but we will no longer be able to do things like
```C#
services.Configure<RazorLSPOptions>(opt => opt.EnableFormatting = false);
```

Reply:
Actually, our `IOptionsMonitor` implementation doesn't care about IConfigureOptions, so we can make it doesn't matter. I'll make it immutable for now.